app/src/main/java/com/kickstarter/ui/activities/MessagesActivity.java

```diff
@@ -45,8 +46,10 @@
   protected @Bind(R.id.messages_recycler_view) RecyclerView recyclerView;
   protected @Bind(R.id.view_pledge_button) Button viewPledgeButton;
 
-  protected @BindString(R.string.backer_modal_pledge_amount_on_pledge_date) String pledgeAmountOnPledgeDateString;
+  protected @BindString(R.string.project_creator_by_creator) String byCreatorString;
+  protected @BindString(R.string.pledge_amount_pledged_on_pledge_date) String pledgeAmountPledgedOnPledgeDateString;
   protected @BindString(R.string.project_view_button) String viewPledgeString;
+  protected @BindString(R.string.Reply_to_user_name) String replyToUserNameString;
```

Comment:
used down in [line 148](https://github.com/kickstarter/android-oss/pull/108/files/854b123c0f727abf672a8577bf03218677562f81#diff-694532fb83d79b4aaa90b074ab9b17eeR148) 😏